### PR TITLE
added new examples for rerank

### PR DESCRIPTION
**Pull request by mahjongmen**

### CODE REVIEW

**Review by lfayoux — DISMISSED** [review]

File: examples/sample_rerank_english_v2_input.json
Comment (leilacc) [inline examples/sample_rerank_english_v2_input.json:1]: could you put these in a subdirectory (like `command_r_samples`?) trying to make this folder easier to navigate & shorten file names

File: notebooks/sagemaker/Deploy rerank english v2.0 model.ipynb
Comment (leilacc) [inline notebooks/sagemaker/Deploy rerank english v2.0 model.ipynb:1]: same for these guys

File: examples/sample_rerank_english_v3_input.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"query": "What is the capital of the United States?", 
+	"return_documents": true, 
+	"rank_fields":["Title","Content"],
```
Comment (leilacc) [inline examples/sample_rerank_english_v3_input.json]: nit: spacing

File: examples/sample_rerank_multilingual_v3_input.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+	"query": "What is the capital of the United States?", 
+	"return_documents": true, 
+	"rank_fields":["Title","Content"],
```
Comment (leilacc) [inline examples/sample_rerank_multilingual_v3_input.json]: nit: spacing

File: notebooks/sagemaker/Deploy rerank english v3.0 model.ipynb [hunk 3]
```diff
@@ -0,0 +1,299 @@
...
+    "import boto3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cohere_package = \"cohere-rerank-english-v2-0-1-8a54b4aa75b733fb87562488368eac4f\"\n",
```
Comment (leilacc) [inline notebooks/sagemaker/Deploy rerank english v3.0 model.ipynb]: don't forget to update this

File: notebooks/sagemaker/Deploy rerank multilingual v3.0 model.ipynb [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+    "import boto3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cohere_package = \"cohere-rerank-multilingual-v2--8b26a507962f3adb98ea9ac44cb70be1\"\n",
```
Comment (leilacc) [inline notebooks/sagemaker/Deploy rerank multilingual v3.0 model.ipynb]: don't forget to update

**Review by leilacc — DISMISSED** [review]

File: notebooks/sagemaker/Deploy rerank english v3.0 model.ipynb (hunk 3 above)
Comment (jkedrzyckii) [reply notebooks/sagemaker/Deploy rerank english v3.0 model.ipynb]: i can't update the package until the model has been configured and subscribed to on Sagemaker. Generally this is the last step before the model is public. Can we publish this and I'll update the model package tomorrow?

File: notebooks/sagemaker/Deploy rerank multilingual v3.0 model.ipynb (hunk 4 above)
Comment (jkedrzyckii) [reply notebooks/sagemaker/Deploy rerank multilingual v3.0 model.ipynb]: i will have to update it tomorrow. This is the last step for the model to be deployed, let's leave as is and i'll make the change tomorrow.

File: examples/sample_rerank_english_v2_input.json
Comment (mahjongmen) [reply examples/sample_rerank_english_v2_input.json:1]: okay

File: notebooks/sagemaker/Deploy rerank english v2.0 model.ipynb
Comment (mahjongmen) [reply notebooks/sagemaker/Deploy rerank english v2.0 model.ipynb:1]: done

File: examples/command_r_samples~ec61408 (Create command_r_samples)
Comment (leilacc) [inline examples/command_r_samples~ec61408 (Create command_r_samples):1]: remove?

File: notebooks/sagemaker/rerank_v2_notebooks/Deploy rerank multilingual v2.0 model.ipynb [hunk 5]
```diff
@@ -138,7 +138,7 @@
    "outputs": [],
    "source": [
     "co = Client(region_name=region)\n",
-    "co.create_endpoint(arn=model_package_arn, endpoint_name=\"cohere-rerank-multilingual\", instance_type=\"ml.g4dn.xlarge\", n_instances=1)\n",
+    "co.create_endpoint(arn=model_package_arn, endpoint_name=\"cohere-rerank-multilingual-v2.0\", instance_type=\"ml.g4dn.xlarge\", n_instances=1)\n",
     "\n",
     "# If the endpoint is already created, you just need to connect to it\n",
     "# co.connect_to_endpoint(endpoint_name=\"cohere-rerank-multilingual\")"
```
Comment (leilacc) [inline notebooks/sagemaker/rerank_v2_notebooks/Deploy rerank multilingual v2.0 model.ipynb:144]: this should get updated to match

**Review by leilacc — DISMISSED** [review]

**Review by leilacc — APPROVED** [review]

**Pull request merged**